### DISCUSSION TITLE
Desperate to get GitHub going for use teaching computer science college classes

### DISCUSSION THREAD

**Discussion by geoffreystaples** [community/community#59910] [GitHub Education]
### Hi everyone,

I have only used github as a version control system for programming projects where I was a programmer checking out code, updating it, and returning it to the github repository.

Now, I want to use github to provide Kotlin and Java students with coding examples, assignment instructions and shells, and for the students to submit their code for grading.

I'm having a problem getting setup and going. Does anyone have any suggestions about how I can bootstrap my gitHub expertise? I need to use it in three programming classes this fall.

**Comment by GhostOf0days** [comment]
I recommend checking out how many Carnegie Mellon classes do it. Here's an example: https://github.com/CMU-17-214/s23-rec05. Students then fork that repo and work on the fork.
